test/recipes/20-test_cli_fips.t

```diff
@@ -46,8 +50,24 @@ ok(run(app(['openssl', 'fipsinstall', '-in', 'fipsmodule.cnf', '-module', $fipsm
 $ENV{OPENSSL_CONF_INCLUDE} = abs2rel(curdir());
 $ENV{OPENSSL_CONF} = $fipsconf;
 
-ok(run(app(['openssl', 'provider', '-v', 'fips'])),
-   "provider listing");
+unless ($deprecated) {
+    ok(run(app(['openssl', 'list', '-public-key-methods', '-verbose'])),
+       "provider listing of public key methods");
+}
```

Comment:
This is the bit that got me to wonder what the hell was going on...

Reply:
more than happy to take this out :)